charts/tidb-cluster/values.yaml

```diff
@@ -47,9 +47,20 @@ discovery:
 # if the ConfigMap was not changed.
 enableConfigMapRollout: true
 
+# Whether enable TLS connections between server nodes.
+# When enabled, PD/TiDB/TiKV will use TLS encrypted connections to transfer data between each node,
+# certificates will be generated automatically (if not already present).
+enableTLSServer: false
```

Comment:
This enables TLS encrypted connections in TiDB cluster. For users, no clients are involved. I prefer `cluster` over `server` here.